src/proposal.mligo

```diff
@@ -69,17 +96,13 @@ let check_proposal_limit_reached (config, store : config * storage): storage =
   then (failwith("MAX_PROPOSALS_REACHED") : storage)
   else store
 
-let freeze_on_ledger (tokens, addr, ledger, total_supply, frozen_token_id, governance_token : nat * address * ledger * total_supply * token_id * governance_token)
-    : (operation * ledger * total_supply) =
+let freeze_on_ledger (tokens, addr, governance_token : nat * address * governance_token)
```

Comment:
2 suggestions:

1. rename this to `lock_governance_tokens`, since there is no longer a ledger to freeze on and all that this function does now is indeed lock governance tokens into the DAO's account.
2. nitpick: return `operation list` instead (with the only element), it will make this cleaner to use in its only call site

Reply:
Renamed this as well as the `unlock_governance_tokens`